x/poolmanager/types/keys.go

```diff
@@ -31,9 +30,6 @@ func FormatModuleRouteKey(poolId uint64) []byte {
 // ParseModuleRouteFromBz parses the raw bytes into ModuleRoute.
 // Returns error if fails to parse or if the bytes are empty.
 func ParseModuleRouteFromBz(bz []byte) (ModuleRoute, error) {
-	if len(bz) == 0 {
```

Comment:
Note: bz of length zero is expected because when we serialize `ModuleRoute` that has `PoolType` `Balancer` that is equal to `zero`, the serialization ends up being empty bytes.